.github/workflows/test.yml

```diff
@@ -33,12 +56,13 @@ jobs:
         with:
           name: k-framework
           authToken: '${{ secrets.CACHIX_PUBLIC_TOKEN }}'
-          skipPush: true
```

Comment:
I think we definitely don't need to push to cachix in this job, since we're not yet sure if the tests pass. We only need to push to cache in the `master-push.yml` job.

Reply:
yeah, I agree.

Reply:
Reverted.